src/styles/main.scss

```diff
@@ -1 +1,31 @@
-// put code here
+@import "imported";
```

Comment:
look, you don't need to hardcode this imports. There is a better way to do this, just make separate `watch.scss` file, and add to current `main.scss` file all imports, so this file can be just:
```scss
@import "watch";
@import "variables";
@import "animation";
@import "normalize";
@import "mix"
```
and create a `blocks` and `utils` dir's to separate them